k6/models/programs.js

```diff
@@ -125,4 +125,11 @@ export default class ProgramsModel {
     }
     return statusCount;
   }
+
+  nedbankCronJob() {
+    const url = `${baseUrl}api/financial-service-providers/nedbank`;
+    const res = http.patch(url);
+    console.log(res.status);
+    return res;
```

Comment:
```suggestion
    const res = http.patch(url);
    return res;
```
This console.log was probably left there fro testing?